### PR TITLE
Illusive cef connector

### PR DESCRIPTION
**Pull request by liadga**
Illusive cef connector + workbooks

### CODE REVIEW

File: DataConnectors/illusive Attack Management System.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
```
Comment (preetikr) [inline DataConnectors/illusive Attack Management System.json:55]: Please add a link to Illusive's website for information on Illusive and what data they'll see.

**Comment by preetikr** [comment]
Please resolve conflicts as well

File: DataConnectors/illusive Attack Management System.json (hunk 1 above)
Comment (liadga) [reply DataConnectors/illusive Attack Management System.json:55]: Thank you @preetikr 
Those are the instructions steps how to connect the our connector. We don't have any additional info in our website on the data we send via CEF (or how to use it). Do you expect just general link to our website or something else?

Comment (preetikr) [reply DataConnectors/illusive Attack Management System.json:55]: Yes, a general link would be great. Thanks.

Comment (preetikr) [reply DataConnectors/illusive Attack Management System.json:55]: Yes, a general link would be great. Thanks.

File: Logos/illusiveAttackManagementSystem.svg [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="528px" height="180px" viewBox="0 0 528 180" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (preetikr) [inline Logos/illusiveAttackManagementSystem.svg:2]: Please remove xmlns:xlink="http://www.w3.org/1999/xlink" as it interferes with rendering logo on Azure portal.

File: Workbooks/Images/Logos/illusive_logo_workbook.svg [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="126px" height="180px" viewBox="0 0 126 180" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (preetikr) [inline Workbooks/Images/Logos/illusive_logo_workbook.svg:2]: Please remove xmlns:xlink="http://www.w3.org/1999/xlink" as it interferes with rendering logo on Azure portal.

File: DataConnectors/illusive Attack Management System.json (hunk 1 above)
Comment (liadga) [reply DataConnectors/illusive Attack Management System.json:55]: Fixed

File: Logos/illusiveAttackManagementSystem.svg (hunk 2 above)
Comment (liadga) [reply Logos/illusiveAttackManagementSystem.svg:2]: Fixed

File: Workbooks/Images/Logos/illusive_logo_workbook.svg (hunk 3 above)
Comment (liadga) [reply Workbooks/Images/Logos/illusive_logo_workbook.svg]: Fixed

**Comment by liadga** [comment]
@preetikr Thank you,

**Pull request merged**